clickhouse-admin/types/versions/src/initial/timeseries.rs

```diff
@@ -0,0 +1,97 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Timeseries types shared by ClickHouse Admin Server and Single APIs.
```

Comment:
This is... not quite right? The timeseries are collected from the `system` database.

In any case. I'm not sure why cluade code decided they should have their own module 😄 I would move all of this to server.rs, as that's the component we're targeting with these.

Reply:
Sure!